03-first-task.Rmd

```diff
@@ -308,12 +308,12 @@ WDL is built to make use of Docker as it makes handling software dependencies mu
 
 * You may not have permission to install software if you are using an institute HPC or other shared resource
 
-When you run a WDL task that has a `docker` runtime attribute, your task will be executed in a Docker container sandbox environment. This container sandbox is derived from a template called a Docker image, which packages installed software in a special file system. This is one of the main features of a Docker image -- because a Docker image packages the software you need, you can skip much of the installation and dependency issues associated with using new software, and because you take actions within a Docker container sandbox, it's unlikely for you to "mess up" your main system's files. Although a Docker container is, strictly speaking, not the same as a virtual machine, it is helpful to think of it as one if you are new to Docker. Docker containers are managed by Docker Engine, and the official Docker GUI is called Docker Desktop.
+When you run a WDL task that has a `docker` runtime attribute, your task will be executed in a Docker container sandbox environment. This container sandbox is derived from a template called a *Docker image*, which packages installed software in a special file system. This is one of the main features of a Docker image -- because a Docker image packages the software you need, you can skip much of the installation and dependency issues associated with using new software. Because you take actions within a Docker container sandbox, it's unlikely for you to "mess up" your main system's files. Although a Docker container is, strictly speaking, not the same as a virtual machine, it is helpful to think of it as one if you are new to Docker. Docker containers are managed by Docker Engine/Apptainer, and the official Docker GUI is called Docker Desktop.
```

Comment:
Maybe we could rework the last sentence a bit for clarity?

> Docker containers are typically managed by Docker Engine, and the official Docker GUI is called Docker Desktop. However, some institutes (especially those using HPCs) use different software for running Docker containers without directly using Docker Engine. Popular container software includes Apptainer, Singularity, and Podman.

If Apptainer is what the Fred Hutch HPC uses, it might be worth linking this: https://apptainer.org/docs/user/main/docker_and_oci.html#best-practices-for-docker-apptainer-compatibility

Reply:
This is the FH page on using Apptainer: https://sciwiki.fredhutch.org/hdc/workflows/running/on_gizmo/#using-apptainer